solr/modules/ltr/src/java/org/apache/solr/ltr/response/transform/LTRFeatureLoggerTransformerFactory.java

```diff
@@ -362,9 +363,13 @@ private void implTransform(SolrDocument doc, int docid, Float score) throws IOEx
       LTRScoringQuery rerankingQuery = rerankingQueries[0];
       LTRScoringQuery.ModelWeight rerankingModelWeight = modelWeights[0];
       for (int i = 1; i < rerankingQueries.length; i++) {
-        if (((LTRInterleavingScoringQuery) rerankingQueriesFromContext[i])
-            .getPickedInterleavingDocIds()
-            .contains(docid)) {
+        Set<Integer> pickedInterleavingDocIds =
+            ((LTRInterleavingScoringQuery) rerankingQueriesFromContext[i])
+                .getPickedInterleavingDocIds();
+        if (pickedInterleavingDocIds == null) {
+          return;
+        }
+        if (pickedInterleavingDocIds.contains(docid)) {
```

Comment:
I wonder if this could be combined here? Will add a commit to explore, feel free to revert.
```suggestion
        if (pickedInterleavingDocIds != null &&
            pickedInterleavingDocIds.contains(docid)) {
```

Reply:
> ... Will add a commit to explore, feel free to revert. ...

https://github.com/apache/solr/pull/937/commits/c7f194fc58a599616cb822ce2c291cae7f693fc8 is the commit.